src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/Views/Shared/_Layout.cshtml

```diff
@@ -10,48 +10,32 @@
         <link rel="stylesheet" href="~/css/site.css" />
     </environment>
     <environment exclude="Development">
-        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.7/css/bootstrap.min.css"
+        <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/4.1.1/css/bootstrap.min.css"
               asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
               asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
         <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
     </environment>
 </head>
 <body>
-    <nav class="navbar navbar-inverse navbar-fixed-top">
-        <div class="container">
-            <div class="navbar-header">
-                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                    <span class="sr-only">Toggle navigation</span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                    <span class="icon-bar"></span>
-                </button>
-                <a asp-area="" asp-controller="Home" asp-action="Index" class="navbar-brand">Company.WebApplication1</a>
-            </div>
-            <div class="navbar-collapse collapse">
-                <ul class="nav navbar-nav">
-                    <li><a asp-area="" asp-controller="Home" asp-action="Index">Home</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                    <li><a asp-area="" asp-controller="Home" asp-action="Contact">Contact</a></li>
-                </ul>
-@*#if (IndividualAuth || OrganizationalAuth)
-                <partial name="_LoginPartial" />
-#elseif (WindowsAuth)
-                <p class="nav navbar-text navbar-right">Hello, @User.Identity.Name!</p>
-#endif*@
-            </div>
-        </div>
-    </nav>
-
-    <partial name="_CookieConsentPartial" />
-
-    <div class="container body-content">
-        @RenderBody()
-        <hr />
-        <footer>
-            <p>&copy; copyrightYear - Company.WebApplication1</p>
-        </footer>
+    <div class="d-flex flex-column flex-md-row align-items-center p-3 px-md-4 mb-3 bg-white border-bottom box-shadow">
+        <h5 class="my-0 mr-md-auto font-weight-normal"><a class="text-dark" asp-area="" asp-controller="Home" asp-action="Index">Company.WebApplication1</a></h5>
+        <nav class="my-2 my-md-0 mr-md-3">
+            <a class="p-2 text-dark" asp-area="" asp-controller="Home" asp-action="Index">Page1</a>
+            <a class="p-2 text-dark" asp-area="" asp-controller="Home" asp-action="Index">Page2</a>
+        </nav>
+    </div>
```

Comment:
Can the above be written using IMO more commonly used comonents based on header/nav tags?
```html
<header>
  <nav class="navbar navbar-expand navbar-light flex-column flex-md-row bg-white border-bottom box-shadow">
    <a class="navbar-brand" asp-area="" asp-controller="Home" asp-action="Index">Company.WebApplication1</a>
    <ul class="navbar-nav ml-md-auto">
        <li class="nav-item">
          <a class="nav-link" asp-area="" asp-controller="Home" asp-action="Index">Page1</a>
        </li>
        <li class="nav-item">
          <a class="nav-link" asp-area="" asp-controller="Home" asp-action="Index">Page2</a>
        </li>
      </ul>
  </nav>
</header>
```
https://codepen.io/blazejewicz/pen/bjpWKR